.github/workflows/combine-prs.yml

```diff
@@ -0,0 +1,152 @@
+name: 'Combine PRs'
```

Comment:
I'd maybe call this "Combine dependabot PRs" to be properly specific. I know you can override the branchPrefix, as it is now, but it might be an idea to remove that input and keep that as a static value. However, dependabot might change the branch prefix. Six and two threes